src/components/TradeWidget/TxMessage.tsx

```diff
@@ -1,21 +1,98 @@
-import React from 'react'
+import React, { useMemo } from 'react'
 import { formatTimeInHours } from 'utils'
 import { useFormContext } from 'react-hook-form'
 import { TokenDetails } from 'types'
 import styled from 'styled-components'
 import { TradeFormData } from '.'
 import { displayTokenSymbolOrLink } from 'utils/display'
+import { usePriceEstimationInOwl } from 'hooks/usePriceEstimation'
+import BigNumber from 'bignumber.js'
+import { ZERO_BIG_NUMBER } from 'const'
+
+import alertIcon from 'assets/img/alert.svg'
+import { useGasPrice } from 'hooks/useGasPrice'
 
 interface TxMessageProps {
   sellToken: TokenDetails
   receiveToken: TokenDetails
+  networkId: number
 }
 
 const TxMessageWrapper = styled.div`
   padding: 1em;
 `
 
-export const TxMessage: React.FC<TxMessageProps> = ({ sellToken, receiveToken }) => {
+const Warning = styled.p`
+  background: beige;
+  padding: 0.5em;
+  box-shadow: inset 0 0 7px 3px #cc262647;
+  border-radius: 0.3em;
+  display: flex;
+
+  .alert {
+    min-width: 2.5em;
+  }
+`
+
+const DEFAULT_GASP_RICE = 40e9 // 40 Gwei
```

Comment:
should we move it to the constants?
Why do we need a default?

Reply:
In case both gasStation and `eth_gasPrice` fail
I want to have some kind of lowVolume estimate even then

Also don't think this default should be app wide, unless we really need an app wide gasPrice
This one is good for estimates, but app-wide we better leave it to provider on each tx.
So as not to encourage app-wide use, let it stay local.